packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -184,6 +185,17 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
       versionMetadata = extractVersionMetadata(providedClientVersion);
     }
 
+    if (!this.hasLoggedBrowserSerial) {
+      const browserSerial = getBrowserSerial();
+      this.logger.log(
+        CALL_DIAGNOSTIC_LOG_IDENTIFIER,
+        `CallDiagnosticMetrics: @createClientEventObjectInMeeting => collected browser data`,
+        JSON.stringify(browserSerial)
```

Comment:
We try to save the userAgent data, but we suspect there is something going wrong with the process of accessing it. The purpose of this PR is to gain some further insight as to why it might be happening. And yeah, we could stringify it in the function, we just decided to do it here

Reply:
oh woops, I'll put the function call inline, i misread what you wrote about that part